clustermesh-apiserver/main.go

```diff
@@ -68,6 +68,15 @@ var (
 		Use:   "clustermesh-apiserver",
 		Short: "Run the ClusterMesh apiserver",
 		Run: func(cmd *cobra.Command, args []string) {
+			// Open socket for using gops to get stacktraces of the agent.
+			addr := fmt.Sprintf("127.0.0.1:%d", viper.GetInt(option.GopsPort))
+			addrField := logrus.Fields{"address": addr}
+			if err := gops.Listen(gops.Options{Addr: addr}); err != nil {
+				log.WithError(err).WithFields(addrField).Fatal("Cannot start gops server")
+			} else {
+				log.WithFields(addrField).Info("Started gops server")
```

Comment:
Nit: `else` branch here is not necessary given the `Fatal` above.

Reply:
fixed